tasks_management/gql_queries.py

```diff
@@ -34,6 +36,11 @@ def _convert_to_serializable_json(entity):
 
     return json.dumps(converted_dict)
 
+def is_task_triage(user):
+    return user.has_perms(TasksManagementConfig.gql_task_group_create_perms) and \
+           user.has_perms(TasksManagementConfig.gql_task_group_search_perms) and \
+           user.has_perms(TasksManagementConfig.gql_task_group_update_perms) and \
+           user.has_perms(TasksManagementConfig.gql_task_group_delete_perms)
```

Comment:
You can do this in one call by combining all permissions into one array. All the gql_task_group_ fields are arrays and you have to concatenate them.

Reply:
done